api/app/src/providers/withings.ts

```diff
@@ -90,6 +91,10 @@ export class Withings extends Provider implements OAuth2 {
     return JSON.stringify(response.data.body);
   }
 
+  async revokeProviderAccess(connectedUser: ConnectedUser) {
+    return this.oauth.revokeWithNoOauth(connectedUser);
+  }
+
```

Comment:
Withings had an enpoint but it requires the withings user id in order to deauthenticate which is weird. So since we have no way of getting that thats why its included here

Reply:
I *think* withings gives us the userid when we get the access token https://developer.withings.com/api-reference/#operation/oauth2-getaccesstoken